Management_System/Management_System/Context/DataGenerator.cs

```diff
@@ -0,0 +1,153 @@
+﻿using Bogus;
+using ManagementSystem.Entities;
+using ManagementSystem.Enums;
+using System.Collections.Generic;
+
+namespace ManagementSystem.Context
+{
```

Comment:
Better to use short form of namespaces since you use .Net 8 `namespace ManagementSystem.Context;`